CodeStarterCamp_Week2/main.swift

```diff
@@ -6,3 +6,78 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
+import Foundation
+
+let lotto: Set<Int> = getlotto()
+
+func getlotto() -> Set<Int>{
```

Comment:
```suggestion
func getLotto() -> Set<Int> {
```

컨벤션에 따르면 단어가 바뀌는 시점에 대문자로 변경, 함수 정의부 중괄호 앞 공백이 필요할 것 같습니다~